eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -194,12 +194,12 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
       goto(WAIT_FOR_OPEN_CHANNEL) using DATA_WAIT_FOR_OPEN_CHANNEL(inputFundee)
 
     case Event(INPUT_RESTORED(data), _) =>
-      log.info(s"restoring channel channelId=${data.channelId}")
+      log.info("restoring channel")
```

Comment:
What is the reasoning for removing the channel ID from this log? It sounds very useful to have the ID there

Reply:
It's already there because of the MDC mechanism.

Reply:
Channel logs are indexed by `nodeId` and `channelId` by default.

This is what it looks like:

```
2020-11-30 09:13:33,394 INFO  f.a.e.i.Peer CON n:020686088612cb34dc1842fcf2d709d47f81843849fe6baeb625aa256ced930ceb c:7f7eba269169d637ca5061e436e3f1bc5e17cdae6b424a5860ed74d150a95112 - restoring channel
```

Reply:
nice. thanks